lib/ret_web/controllers/api/v1/project_assets_controller.ex

```diff
@@ -15,7 +15,26 @@ defmodule RetWeb.Api.V1.ProjectAssetsController do
     end
   end
 
-  def create(conn, %{"id" => project_sid, "asset" => params}) do
+  def create(conn, %{"project_id" => project_sid, "asset_id" => asset_sid}) do
+    account = conn |> Guardian.Plug.current_resource()
+
+    case Project.project_by_sid_for_account(account, project_sid) do
```

Comment:
you can refactor this into `with` to reduce nesting, etc. same below

Reply:
you have a few other `case` statements like this that are less nested but will likely be better if they used `with` macro